README.md

```diff
@@ -684,7 +721,82 @@ usr/src/uts/intel/os/name_to_major
 usr/src/uts/intel/os/minor_perm
 ```
 
-## Testing and Things to Think About
+# Contributing
```

Comment:
If you *like*, this section could be moved to a separate CONTRIBUTING.md file. GitHub will then provide an info box and a link to that file in the UI for creating a PR.

Reply:
I don't think this is really necessary. Everyone needs to read the whole thing anyway.